test/piezo.js

```diff
@@ -781,10 +807,11 @@ exports["Piezo - I2C Backpack"] = {
   tone: function(test) {
     test.expect(3);
 
-    this.piezo.tone(262, 1000);
+    // C4 = 262Hz = 1908μs duty cycle (half-period)
+    this.piezo.tone(1908, 1000);
```

Comment:
I updated this test because it could be misleading and imply that `tone` was supposed to be called with a frequency (since 262Hz is a familiar frequency C4, used elsewhere in these tests).